### DISCUSSION TITLE
Ventoy can't load images on an old laptop

### DISCUSSION THREAD

**Discussion by mvasiliiv** [ventoy/Ventoy#1187] [General]
Dear developers, I really love your product and constantly use it, but recently I discovered a laptop on which loading from images does not work, I get into the ventoy menu without problems, but after I select any iso, there is no loading from it, instead I see a black screen and a blinking cursor and so on indefinitely. I recorded a video of the process of recording a usb flash drive and downloading from it on this laptop (acer extensa 5620). I tried different usb flash drives, the result is the same. At the same time, using another program to record a multi-boot flash drive on the same laptop - everything works, for this reason I understand that the problem is not in the laptop itself. I want to understand what is the peculiarity and what prevents ventoy from working correctly. Perhaps you can fix this problem and in the next update everything will work as expected. If you need any additional data from me, I am ready to provide everything, and now I am sending you a link to the video
https://disk.yandex.by/i/OzzVU2X8LKzDsQ

**Comment by steve6375** [comment]
Does Easy2Boot grub4dos (and agFM grub2) also fail in same way?

**Comment by mvasiliiv** [comment]
I didn't understand a little what you were asking, perhaps because I don't understand English and use a translator

**Reply by steve6375** [reply]
https://www.easy2boot.xyz  - please test

**Reply by mvasiliiv** [reply]
Now I am preparing the media using this program (easy2boot) and will immediately write about the result

**Comment by mvasiliiv** [comment]
Everything works, but I'm asking about Ventoy, maybe you know the solution to the problem?

**Reply by steve6375** [reply]
did you also test the agFM menu which is based on grub2 (same as Ventoy)?
![image](https://user-images.githubusercontent.com/1469350/139574851-7e600640-f2f5-4725-b792-d05e949eab23.png)

**Reply by mvasiliiv** [reply]
No, I just checked the menu item "Install from ISO"
Now I will test what you said

**Reply by mvasiliiv** [reply]
I have an item missing from the menu - (agFM)
What could be the matter and how to include this item or make it appear?

**Reply by steve6375** [reply]
Did you install with Win10 OS?

**Reply by mvasiliiv** [reply]
No, I don't have the ability to burn a usb flash drive on windows 10, I'm using windows 8.1, is that the problem?

**Reply by steve6375** [reply]
yes. Need win10 for USB Flash drives (2 partitions used).

**Reply by steve6375** [reply]
With Ventoy - did you try  ctrl+i, ctrl+w and ctrl+r  hotkey first?

**Reply by mvasiliiv** [reply]
That is, I will not be able to check the download in agFM mode and we will not be able to solve the issue with downloading from iso to ventoy

**Reply by mvasiliiv** [reply]
No, I haven't used such keyboard shortcuts, but what do they give and when do I need to press them?

**Reply by steve6375** [reply]
Press Ctrl+h or h for help.
1. Ventoy menu
2. Press Ctrl key
3. Select ISO - press ENTER key

**Reply by steve6375** [reply]
![image](https://user-images.githubusercontent.com/1469350/139575323-fbd41a2f-088b-44ce-b4f3-adc80fa42474.png)

**Reply by mvasiliiv** [reply]
Now I am overwriting the usb flash drive with ventoy and I will try

**Reply by mvasiliiv** [reply]
You're a genius!
Thank you very much!
It was necessary to press ctrl+i and in this mode ventoy was able to boot from iso without any problems
The question can be closed!

**Reply by steve6375** [reply]
well ctrl-i is for debug only - it should work without ctrl-i ...

**Reply by mvasiliiv** [reply]
And then what should I do, since it works in this mode, maybe there are thoughts

**Reply by mvasiliiv** [reply]
Also checked it works in the mode ctrl+w

**Reply by mvasiliiv** [reply]
What would be the right thing to do in such a situation?

**Reply by mvasiliiv** [reply]
I hurried to write that it works in ctrl+w mode, the download is happening, but the installation is not possible, does not see the device driver, there is no ssd / hdd in the list of devices (this is already when trying to install windows 7)

**Reply by steve6375** [reply]
Wait for developer LongPanda to reply...

**Reply by mvasiliiv** [reply]
Got it, thanks. I was overjoyed in vain, I can't install it in ctrl+i mode either, the download from iso happens, but it doesn't go any further. I'm waiting for the developer then, I hope he will appear soon.

**Reply by longPanda** [reply]
> Wait for developer LongPanda to reply...

What happened? Did I say anything? Why call me!

**Comment by ventoy** [comment]
1. Only Windows ISO has this problem?  Try a linux iso (e.g. Ubuntu)
2. Please do some debug.  Press `c` to enter grub shell and run `vtdebug on` and then press `ESC` to return and then select the ISO file and boot.

**Reply by mvasiliiv** [reply]
1. linux booted without problems.
2. I did this operation, it did not give results, windows images also do not load

**Reply by ventoy** [reply]
Can you make another video about the process 2?

**Reply by mvasiliiv** [reply]
Yes, now I'll write everything down, maybe I'm doing something wrong

**Reply by mvasiliiv** [reply]
Here you are, if I'm doing something wrong, tell me how it should be
https://disk.yandex.by/i/qqxOUF2_7IzIeQ

**Reply by ventoy** [reply]
Can the same windows7_32bit iso boot OK if you plug the same USB to another computer?

**Reply by ventoy** [reply]
Can you test another windows iso file(e.g. Windows 10 64bit) ?

**Reply by mvasiliiv** [reply]
Yes, maybe the image is working like my other images. Now I will do the same with Windows 10.
And what about vtdebug, no more action is required?

**Reply by mvasiliiv** [reply]
I checked on the windows 10 image, the result is the same, a blinking cursor on a black screen

**Reply by mvasiliiv** [reply]
Dear developer, please do not disregard my question, I think you just like I want to understand this situation and do everything to make the program truly universal

**Reply by ventoy** [reply]
Currently I have no idea about this issue.
I can give a workaround to make ctrl+i avaliable for standard windows iso file in a few days.

**Reply by mvasiliiv** [reply]
Any working solution will do!  I need to wait for the next update and then it will be possible to use ctrl + i in such situations and during installation there will be no problems with the drive driver?

**Reply by mvasiliiv** [reply]
If it's not difficult, answer my last question

**Comment by mvasiliiv** [comment]
Version 1.0.58 did not solve my problem, I hope that there will be a solution in the future

**Comment by ventoy** [comment]
@mvasiliiv 
Here is a workaround for `Ctrl+i`
1. Download the following 2 files and put them in the root of Ventoy partition where ISO file locates.
https://github.com/ventoy/vtoydump/blob/master/bin/NT6/64/vtoydump.exe  --> save as vtoydump64.exe
https://github.com/ventoy/vtoydump/blob/master/bin/NT6/32/vtoydump.exe  --> save as vtoydump32.exe

2. Use `Ctrl+i` to boot the Windows ISO file. When the setup screen popup, don't rush to click `Next` button, but press `Shift+F10` to launch cmd.exe and run  `C:\vtoydump64.exe  -m` after that click `Next` to install as normal.
Here  `C:\vtoydump64.exe  -m` is an example, the exe file may be in  C:\ or D:\ or E:\ or ......  
So the command may be  `C:\vtoydump64.exe  -m` or `D:\vtoydump64.exe  -m` or `E:\vtoydump64.exe  -m` or ......
Besides, if the Windows ISO is 32bit, then you should use  `vtoydump32.exe`

**Reply by mvasiliiv** [reply]
Super, everything worked out. But now there is one unresolved problem, how to run WinPE on the same old laptop, if you use ctrl+i, the menu loads, but after selecting the WinPE version, an unsuccessful attempt to boot occurs and the laptop goes into reboot

**Reply by mvasiliiv** [reply]
Before loading WinPE, I can't use the command line, it's not called with shift+F10
What kind of workaround can help here?

**Reply by ventoy** [reply]
Which PE did you test? Are you sure the PE can boot fine in your laptop? If you burn the PE iso to the same USB with Rufus can it boot OK?

**Reply by ventoy** [reply]
Some small WinPE only contains a WIM file. You can copy the WIM file to Ventoy usb and boot with wim boot plugin.
https://www.ventoy.net/en/plugin_wimboot.html

**Reply by mvasiliiv** [reply]
Yes, WinPE Strelec loads on this laptop without any problems if it is recorded by other software. Now I started in ctrl+i mode and chose a different version of PE, there are several options on the Strelec build, Win10 64, Win10 32, Win8 32 and Win8 for old computers, only Win8 32 and Win10 32 started, but after downloading it does not work quite correctly, it does not find the path to the folder with programs and utilities, but this is probably not a question for you, but for the creator of WinPE.

I extracted the Win8 32 wim file and tried to run it using the wimboot plugin, unfortunately it was also unsuccessful and in ctrl+i mode as well, I will continue to write to the Strelec developer. If you still have any useful tips for me, I will be glad to read them, thank you

**Reply by ventoy** [reply]
For  WinPE Strelec  Win10 64, it boot OK when you burn the ISO with Rufus and can not boot with Ventoy, right?
Can you make a photo about the screen with problem?

**Reply by mvasiliiv** [reply]
That's right, the recorded flash drive with the help of Rufus loads without problems, any version, including Win 10 64.
And if you load with Ventoy, it starts only in ctrl+i mode and after selecting Win8 32 or Win 10 32, an error will be displayed on the screen, attached screenshots
http://lostpix.com/img/2021-11/10/1sjxp5wkig1em8dyk8q561ltp.png
http://lostpix.com/img/2021-11/10/6h1t8j3uzwhl8pryu4wxs8u92.png
http://lostpix.com/img/2021-11/10/qycae8ily2i9u4xhb9erq4bkq.png
http://lostpix.com/img/2021-11/10/z3nk1e2pg2te0542d2l7ps9ox.png
http://lostpix.com/img/2021-11/10/p5rqe8g8krypiev0x10su1vlb.png

**Reply by steve6375** [reply]
When you boot to Strelec desktop, can you see the USB drive in Windows Explorer in WinPE?

**Reply by mvasiliiv** [reply]
Yes, I see.  At least I've always seen before, right now I'm not at my laptop, I'm writing from my smartphone, I urgently needed to leave

**Reply by mvasiliiv** [reply]
longpanda, what do you think?

**Reply by mvasiliiv** [reply]
steve, why are you asking this?  What does it give us?

**Reply by steve6375** [reply]
Some versions of Strelec WinPE have missing USB drivers. If you cannot see the USB drive when WinPE fails to load properly, then it means you have a driver+USb drive+USB port compatibility problem.

**Reply by mvasiliiv** [reply]
@ventoy 
what do you think looking at the screenshots?

**Reply by longPanda** [reply]
> longpanda, what do you think?
What happened? Did I say anything? Why call me!

**Reply by ventoy** [reply]
Please give other screenshots:
1. The problem when you boot directly and select Win10 64 PE 
2. The problem when you use ctrl+i to boot and select Win10 64 PE 
3. Extract the Win10 64 wim file and boot with wimboot plugin

**Reply by ventoy** [reply]
How to contact  Strelec developer?

**Reply by mvasiliiv** [reply]
The Strelec developer has a telegram (https://t.me/StrelecSergei), but he does not want to solve such a problem, since he recommends using other ways to write his WinPE, in which 100% performance is guaranteed.
In the screenshots below, I showed everything in detail and described everything in text, everything that you asked me to do.

**Reply by mvasiliiv** [reply]
http://lostpix.com/img/2021-11/11/4wtxf4apnr7feyrb469fp9r35.png
http://lostpix.com/img/2021-11/11/poq3eov19i62jlf1hk637zx65.png
http://lostpix.com/img/2021-11/11/d4l6y1ketptfrnbjxmouzjqsf.png
http://lostpix.com/img/2021-11/11/fgbaj5mfsdqxw6g2bxfc74plh.png
http://lostpix.com/img/2021-11/11/abrqtxw1yw0t9fcwzyr7qtwvq.png
http://lostpix.com/img/2021-11/11/pb3xb4esn2edp2d91ja36zqnw.png
http://lostpix.com/img/2021-11/11/87ftsw9upvj1ewri17a561d6s.png

**Comment by ventoy** [comment]
```
only Win8 32 and Win10 32 started, but after downloading it does not work quite correctly
```

When you use ctrl+i to boot Win8 32 /Win10 32,  launch cmd.exe and run `vtoydump32.exe`  as my previous post (e.g. `C:\vtoydump32.exe  -m`)  
After that click `Reload shotcuts program` icon in the desktop, after it finish reload, it should work OK.
![image](https://user-images.githubusercontent.com/59477474/141230393-da62f7aa-2628-49b2-90ff-08293464f12b.png)

**Reply by mvasiliiv** [reply]
When trying to restart by clicking on this shortcut, the same error pops up, since it was not possible to use vtoydump32

**Reply by ventoy** [reply]
Why it was not possible to use vtoydump32?  run  `C:\vtoydump32.exe -v -m` and give me the log.

**Reply by mvasiliiv** [reply]
The screenshot above showed why it was not possible to use
http://lostpix.com/img/2021-11/11/87ftsw9upvj1ewri17a561d6s.png
Now I'm not using my laptop anymore, if necessary, I'll send a report later

**Reply by ventoy** [reply]
OK, I need more screenshots:
1. use ctrl+i to boot into Win10 64 PE, and run `D:\vtoydump64.exe  -v  -m`
2. use ctrl+i to boot into Win10 32 PE, and run `D:\vtoydump32.exe  -v  -m`

Don't forget the `-v` parameter.

But since you already known where the WinPE ISO locate.
What vtoydump.exe does is just help you to mount the WinPE ISO, but you can also mount it by yourself.
Find the WinPE ISO and right click and select the `Mount the image to imdisk` menu to mount it.
After that click the `Reload shotcuts program` on the desktop.

**Reply by mvasiliiv** [reply]
It worked, thanks!
I can't help but ask a question, is it possible to make these workarounds standard so that everything works out of the box, without ctrl+i and vtoydump, is it worth waiting for everything to work right away in the next update?

**Reply by ventoy** [reply]
NO. 
Normally there is no need to do any workaround. I boot this PE in my PC and laptop without any problem.
But it has problem in your old laptop, so ctrl+i is just a workaround for this case like yours.

**Reply by mvasiliiv** [reply]
Thank you so much, together we make ventoy better!

**Comment by olekstomek** [comment]
https://github.com/ventoy/Ventoy/issues/1252
